### PR TITLE
Load 28511

### PR DESCRIPTION
**Pull request by hammalas**

### CODE REVIEW

**Comment by sylbn** [comment]
LGTM

File: templates/deployment-back.yaml [hunk 1]
```diff
@@ -121,6 +121,8 @@ spec:
               value: {{ $value | quote }}
             {{- end }}
             {{- end }}
+            - name: CORS_ALLOWED_ORIGIN_PATTERN
+              value: {{ default "https?://.*.neotys.com.*" (printf "%s%s" (include "nlweb.helpers.getScheme" .) (.Values.services.files.host) | quote) }}
```
Comment (sylbn) [inline templates/deployment-back.yaml:125]: Should be based on Values.services.webapp.host ?

File: templates/deployment-front.yaml [hunk 2]
```diff
@@ -86,6 +86,14 @@ spec:
...
             {{- end }}
             {{- end }}
+            - name: COOKIE_USE_SSL
+            {{- if eq ((include "nlweb.helpers.getScheme" .) | toString) "https://" }}
+              value: "true"
+            {{- else }}
+              value: "false"
+            {{- end }}
+            - name: COOKIE_DOMAIN
+              value: {{ default ".neotys.com" (.Values.services.files.host | quote) }}
```
Comment (sylbn) [inline templates/deployment-front.yaml:96]: Should be based on Values.services.webapp.host ?

**Review by sylbn — COMMENTED** [review]
Values.services.webapp.host should be used instead of Values.services.files.host

File: templates/deployment-back.yaml (hunk 1 above)
Comment (hammalas) [reply templates/deployment-back.yaml]: Yes

File: templates/deployment-front.yaml (hunk 2 above)
Comment (hammalas) [reply templates/deployment-front.yaml]: yes

**Comment by lcharlois-neotys** [comment]
LGTM.

**Pull request merged**